src/OrbitGl/TextRenderer.cpp

```diff
@@ -25,11 +25,11 @@
 #include "OrbitBase/Logging.h"
 #include "OrbitBase/Tracing.h"
 
-typedef struct {
+using vertex_t = struct {
```

Comment:
What about `struct vertex_t { ... };` as usual?

Reply:
Hehe I got distracted by the existing `typedef`. Done.